python/ray/tune/trainable.py

```diff
@@ -436,10 +451,12 @@ def save(self, checkpoint_dir: Optional[str] = None) -> str:
         checkpoint_dir = TrainableUtil.make_checkpoint_dir(
             checkpoint_dir or self.logdir, index=self.iteration
         )
-        checkpoint = self.save_checkpoint(checkpoint_dir)
```

Comment:
I like it to make naming more explicit!
I used to find it confusing to reason about what is dir and what is path and sometimes the codebase uses them interchangeably.